fcp/src/containment/__init__.py

```
@@ -9,10 +10,11 @@
 )
 from containment.oracles import imp_oracle
 from containment.loops import proof_loop
-from containment.mcp_server import mcp
+from containment.mcp.server import mcp
+from containment.mcp.clients.boundary import Boundary
```

Ruff (0.11.9)

`containment.mcp.clients.boundary.Boundary` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)